src/core.js

```diff
@@ -913,6 +913,15 @@ $.extend( $.validator, {
 				valid: true,
 				message: this.defaultMessage( element, "remote" )
 			});
+		},
+
+		// cleans up all forms and elements, removes validator-specific events
+		destroy: function() {
+			this.resetForm();
+
+			$( this.currentForm ).unbind( "submit.validate" );
+			$( this.currentForm ).find(" :input ").unbind( "focusin.validate focusout.validate keyup.validate click.validate invalid-form.validate" );
```

Comment:
None of these events are bound to any inputs directly, they all use event delegation.